src/features/game/types/consumables.ts

```diff
@@ -515,8 +531,17 @@ export const CONSUMABLES: Record<ConsumableName, Consumable> = {
   },
 };
 
-const Juices = getKeys(CONSUMABLES).filter(
-  (name) => CONSUMABLES[name].building === "Smoothie Shack"
+export const CONSUMABLES: Record<ConsumableName, Consumable> = {
+  ...COOKABLES,
+  "Pirate Cake": {
+    name: "Pirate Cake",
+    description: "Great for Pirate themed birthday parties.",
+    experience: 1500,
```

Comment:
in the docs the xp is 3000, is this changed?

![image](https://user-images.githubusercontent.com/94913189/215632967-24795a99-d854-4dff-9115-c26e2ec1df86.png)

Reply:
> in the docs the xp is 3000, is this changed?

Thanks for pointing this out @mGranTY ! Will fix